src/expressions/operations.py

```diff
@@ -111,16 +155,24 @@ def __repr__(self):
         return f"{self.numerator} / {self.denominator}"
     
     def expand(self):
-        expanded1 = self.exp1.expand()
-        expanded2 = self.exp2.expand()
-        if isinstance(expanded1, Sum):
-            return Sum(*[Quotient(e, expanded2).expand()
-                         for e in expanded1.exps])
-        elif isinstance(expanded2, Sum):
-            return Sum(*[Quotient(expanded1, e)
-                         for e in expanded2.exps])
+        expandedNum = self.numerator.expand()
```

Comment:
I like that you changed the variable names so they are more clear what they mean